config/VehicleSettingsSetup.xml

```diff
@@ -95,6 +95,8 @@
 		<Setting classType="AIParameterBooleanSetting" name="selfUnload"/>
 		<!--Unload on first Headland-->
 		<Setting classType="AIParameterBooleanSetting" name="unloadOnFirstHeadland" defaultBool="false" isExpertModeOnly="true"/>
+		<!--UnloadLevel-->
+		<Setting classType="AIParameterSettingList" name="callUnloaderPercent" min="0" max="100" incremental="5" default ="80" unit="4" isExpertModeOnly="false"/>
```

Comment:
min should not be 0 (no idea if the code can handle that case), should be some reasonable value, like 50?

Reply:
Tested and it works perfectly, just like in Autodrive.